mambaSharedFramework/Pantos-Generic Playlist Parsing/Pantos-Generic Tag Validators/EXT_X_KEYValidator.swift

```diff
@@ -36,15 +36,15 @@ class EXT_X_KEYValidator: GenericDictionaryTagValidator {
             
             switch value {
 
-            case HLSEncryptionMethodType.EncryptionMethod.None.rawValue:
+            case EncryptionMethodType.EncryptionMethod.None.rawValue:
```

Comment:
Did you mean to use `PlaylistEncryptionMethodType`. I won't comment on any other prior uses of `*HLS*` that weren't replaced with `*Playlist*`.

Reply:
The rule isn't to replace `HLS` -> `Playlist`. The rule is to make the same make more sense in a Swift world.

This is just a `EncryptionMethodType`. It does not necessarily refer to a Playlist.

If you see any other places that don't make sense call them out.